e2e/test/scenarios/embedding/embedding-dashboard.cy.spec.js

```diff
@@ -758,6 +760,132 @@ describeEE("scenarios > embedding > dashboard appearance", () => {
       cy.get("@previewEmbedSpy").should("have.callCount", 1);
     });
   });
+
+  it("should not rerender the static dashboard with tabs preview unnecessarily (metabase#46378)", () => {
```

Comment:
This takes the base template from the above test but adds tabs to the dashboard.

When a dashboard has tabs, it use to rerender the preview when toggling options in Look and Feel tab. But it seems this has already been fixed in the upstream.